### ISSUE TITLE
Importing list items when creating through studio 

### ISSUE THREAD

**Issue by darthtang** [salesagility/SuiteCRM#713]
Say you were to have lots and lots of values for a type of drop down list of a field type that contains lots of values.

A cool feature i feel would be to have a sort of importer that will take values from a spreadsheet and shove them into the list items.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Importing list items when creating through studio](https://trello.com/c/kJyFEfOt/132-importing-list-items-when-creating-through-studio)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
